.github/workflows/lint.yml

```diff
@@ -14,7 +14,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: xt0rted/markdownlint-problem-matcher@78410d0625df50bd14b8e178749fdb66e07c31e0
       - uses: actions/setup-node@v3
       - run: npm install
       - run: ./node_modules/.bin/markdownlint **/*.md --ignore node_modules
```

Comment:
looksl ike this line is failing? If we don't install `markdownlint` we can run the command with `npx markdownlint **/*.md --ignore node_modules`

Reply:
try this

```
npx --package markdownlint-cli markdownlint **/*.md --ignore node_modules
```